### PR TITLE
Fixed GIF won't load in Gmail #8682

### PR DESCRIPTION
**Pull request by MntRushmore**
## Summary of the problem
This hopefully will make the GIF show up in the email..

## Describe your changes
Updated GIF to Base64 format. (Didn't Work)
Uploaded to CDN (Didn't work)
Trying with Giphy now

### CODE REVIEW

File: app/views/user/subordinate_summary_mailer/weekly.html.erb [hunk 1]
```diff
@@ -4,7 +4,8 @@
   Your direct reports spent <%= render_money(-@subordinate_stats.sum(&:total_cents)) %> in total this week.
 </p>
 
-<img src="https://cloud-p0kq3pldx-hack-club-bot.vercel.app/0giphy.gif" width="50%" alt="raccoon" title="raccoon">
+<img src="data:image/gif;base64,R0lGODlhgAHgAff/AAYDAAcIAA4FAA4JDw8JABAMABEKBRMSAxMSDxQJABUSABkPERkQBBkZEhoXAxsQAB0WFx4bBCAXDyAgGyITAiIgByIhFSUXBiUkDyYcGyYpJScrEigaCCgoGSkcFSozJywoEi0eDi0zFy8lJTAkGDA6LTEwKDMlETM8HzQwGDVCNzYtLjg3LTktJDlDJzpKPzssFzs1Ozs4Hz40LT4+Nj9KLUA1KUBRRUI/KEM1IEM8QEZHP0ZRM0dXTkg7MklHLkpDRks8KExORExXO01dVU9DO09KU1BNNlJCLVNUSVRJRVRdP1ZJP1ZRVlZiX1dSO1hlVllJM1laUllkR1tRRlxYWlxoZ19ZQF9rT2BPPGBvb2JXTmJeY2JiVmRgSWRxVWZvY2dVP2dbVWd2c2eBf2hkamlnUGp3WW1bS25hWW5qb25tVm52aG9bQG9/b3FiVHNv..." width="50%" alt="raccoon" title="raccoon">
```
Comment (garyhtou) [inline app/views/user/subordinate_summary_mailer/weekly.html.erb]: I'm not sure if this is a valid image. I see that the base64 might have been accidentally truncated (with the `...` at the end)

**Review by garyhtou — COMMENTED** [review]
thanks for volunteering to fix this!

**Comment by MntRushmore** [comment]
Let me check it out and redo the encoding.

**Review by sampoder — APPROVED** [review]
Let's give this a go

File: app/views/user/subordinate_summary_mailer/weekly.html.erb [hunk 2]
```diff
@@ -4,7 +4,9 @@
   Your direct reports spent <%= render_money(-@subordinate_stats.sum(&:total_cents)) %> in total this week.
 </p>
 
-<img src="https://cloud-p0kq3pldx-hack-club-bot.vercel.app/0giphy.gif" width="50%" alt="raccoon" title="raccoon">
+
```
Comment (sampoder) [inline app/views/user/subordinate_summary_mailer/weekly.html.erb]:
```suggestion
```

File: app/views/user/subordinate_summary_mailer/weekly.html.erb [hunk 3]
```diff
@@ -4,7 +4,8 @@
   Your direct reports spent <%= render_money(-@subordinate_stats.sum(&:total_cents)) %> in total this week.
 </p>
 
-<img src="https://cloud-p0kq3pldx-hack-club-bot.vercel.app/0giphy.gif" width="50%" alt="raccoon" title="raccoon">
+<img src="https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExejQ0OXB2ajNnanVjaXlvZW9uaHRta2Yycm42NzNpbXR5MjFuYXRuNSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/vRF0wsDE6U2LyK747C/giphy-downsized-large.gif" width="50%" alt="raccoon" title="raccoon">
+
```
Comment (sampoder) [inline app/views/user/subordinate_summary_mailer/weekly.html.erb]:
```suggestion
```

**Pull request merged**